### PR TITLE
add XmlnsPrefix which is in WPF

### PR DESCRIPTION
**Pull request by bmello4688**
Added similar because it is in WPF. Its for Xaml files. Same use as in WPF

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4891) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4891) before we can accept your contribution.<br/><br/>:x: bmello4688 [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4891)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/Avalonia?pullRequest=4891) it.</sub>

File: src/Avalonia.Base/Metadata/XmlnsPrefixAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Avalonia.Metadata
+{
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    public sealed class XmlnsPrefixAttribute : Attribute
```
Comment (MarchingCube) [inline src/Avalonia.Base/Metadata/XmlnsPrefixAttribute.cs:6]: I guess we should also include class level comment to make it clear what this attribute does for the users.

**Review by MarchingCube — COMMENTED** [review]
Thanks for your contribution. I wonder - what is the use case here - tooling, or something else?

**Comment by bmello4688** [comment]
It’s so you can pre-define namespaces and associated namespace key in your Xaml file.

Brian

Brian
________________________________

**Comment by bmello4688** [comment]
Not key prefix

Brian
________________________________

File: src/Avalonia.Base/Metadata/XmlnsPrefixAttribute.cs (hunk 1 above)
Comment (bmello4688) [reply src/Avalonia.Base/Metadata/XmlnsPrefixAttribute.cs:15]: Added class level comment

**Comment by maxkatz6** [comment]
I like idea

**Review by MarchingCube — APPROVED** [review]
I think having the attribute is fine, I guess that would also allow tooling to actually respect this attribute in the future.

**Comment by bmello4688** [comment]
Sorry closed this by accident after approval

**Pull request merged**